.github/workflows/nuget-push.yml

```diff
@@ -43,6 +43,3 @@ jobs:
 
           # NuGet server uri hosting the packages, defaults to https://api.nuget.org
           NUGET_SOURCE: https://api.nuget.org
-
```

Comment:
With this I think you should instead pin the action to my fork of publish-nuget which is Elskom/publish-nuget. But to use it's edits you would have to have it reference the main branch.

Reply:
Thanks! I'll try this pull request first, and if it doesn't work, I'll try your suggestion.

Reply:
I tried using your suggestion, but I can't get it working, and I can't figure it out:
https://github.com/q2ebanking/boa-constrictor/runs/3079018775?check_suite_focus=true

Reply:
I can help you as soon as I get home with a followup pr.